docs/HowToUseSmartFS.md

```diff
@@ -0,0 +1,187 @@
+# How to use SmartFS
+
+SmartFS is FileSystem used for user partition in TizenRT.
+
+SmartFS stands for Sector Mapped Allocation for Really Tiny (SMART) flash.  
+It is a filesystem that has been designed to work primary with small,
+serial NOR type flash parts that are 1M byte to 16M byte in size
```

Comment:
Can we mention what is the least size it supports currently ?

Reply:
Will check